### PR TITLE
Merge feature/result-images to main

### PR DESCRIPTION
**Pull request by yvonneyanng**
- Deprecated the image download function due to inconsistent layout in the screen and the downloaded image.
- Added story ending text.

### CODE REVIEW

File: components/Landing.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default function Landing({ onStart }: LandingProps) {
   }, []);
 
   return (
-    <div className="relative flex flex-col justify-start items-center h-[100dvh] md:h-screen bg-custom">
+    <div className="relative flex flex-col justify-start items-center h-[100dvh] md:h-screen  bg-custom">
```
Comment (kevinyen83) [inline components/Landing.tsx]: Dont forget to run Prettier before pushing the code.

**Review by kevinyen83 — CHANGES_REQUESTED** [review]
Dont forget to run Prettier before pushing the code.

Additionally, since we handle a lot of states within the components, I recommend exploring the use of Redux to simplify and clarify state management.

Note: If your update involves a complex use case (like this one), feel free to request a UAT test from Kaelyn (PM). She will assist in identifying any potential edge cases that may arise.

File: components/Landing.tsx (hunk 1 above)
Comment (yvonneyanng) [reply components/Landing.tsx]: Will focus on the Redux before next review👍

**Comment by kevinyen83** [comment]
LGTM

**Review by kevinyen83 — APPROVED** [review]
LGTM
